csharp/src/Apache.Arrow.Adbc/C/CAdbcDriverImporter.cs

```diff
@@ -331,155 +331,72 @@ public override unsafe UpdateResult ExecuteUpdate()
 
             public override object GetValue(IArrowArray arrowArray, Field field, int index)
```

Comment:
Is there any real difference between this and the BigQueryStatement implementaton now? Could a base implementation be moved to AdbcStatement?